### PR TITLE
notary 0.2 changelog

### PR DESCRIPTION
**Pull request by riyazdf**
I've started a `CHANGELOG.md` file to track release notes.  I'm not as familiar with notary versioning prior to 0.2 (and I might have missed notes for 0.2 as well), so please feel free to jump in and add whatever notes are appropriate.

Signed-off-by: Riyaz Faizullabhoy <riyaz.faizullabhoy@docker.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+## 0.2 (2016-02-22)
+
++ Add support for delegation roles in `notary` server and client
++ Add `notary CLI` commands for managing delegation roles: `notary delegation`
++ Enhance `notary CLI` commands for adding targets to delegation roles: `notary add --roles`
++ Add remote snapshot key management functionality, such that `notary-server` can manage snapshot keys and sign snapshots for a repository
```
Comment (cyli) [inline CHANGELOG.md]: Apologies, this is probably not going to make it into 0.2 :|

Comment (riyazdf) [inline CHANGELOG.md]: That's no problem, I'll update

**Comment by riyazdf** [comment]
removed WIP, made the changelog point to release notes and use the same text for `v0.2`

I think it'd be nice to have this because the releases page is not as visible when perusing the github repository

**Comment by cyli** [comment]
LGTM pending CI!

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
++ Update `docker-compose` configuration to use official mariadb image
+    + deprecate `notarymysql`
+    + default to using a volume for `data` directory
+    + use separate databases for `notary-server` and `notary-signer` with separate users
++ Add `notary CLI` command for changing private key passphrases: `notary key passwd`
++ Enhance `notary CLI` commands for importing and exporting keys
++ Change default `notary CLI` log level to fatal, introduce new verbose (error-level) and debug-level settings
++ Store roles as PEM headers in private keys, incompatible with previous notary v0.1 key format
+    + No longer store keys as `<KEY_ID>_role.key`, instead store as `<KEY_ID>.key`; new private keys from new notary clients will crash old notary clients
++ Support logging as JSON format
```
Comment (endophage) [inline CHANGELOG.md]: on server and signer.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
++ Support mutual TLS between notary client and notary server
```
Comment (endophage) [inline CHANGELOG.md]: We added this?

Comment (riyazdf) [inline CHANGELOG.md]: merged PR here:  https://github.com/docker/notary/pull/545

File: CHANGELOG.md (hunk 2 above)
Comment (riyazdf) [inline CHANGELOG.md]: done!

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    + deprecate `notarymysql`
+    + default to using a volume for `data` directory
+    + use separate databases for `notary-server` and `notary-signer` with separate users
++ Add `notary CLI` command for changing private key passphrases: `notary key passwd`
++ Enhance `notary CLI` commands for importing and exporting keys
++ Change default `notary CLI` log level to fatal, introduce new verbose (error-level) and debug-level settings
++ Store roles as PEM headers in private keys, incompatible with previous notary v0.1 key format
+    + No longer store keys as `<KEY_ID>_role.key`, instead store as `<KEY_ID>.key`; new private keys from new notary clients will crash old notary clients
++ Support logging as JSON format on server and signer
++ Support mutual TLS between notary client and notary server
```
Comment (HuKeping) [inline CHANGELOG.md]: Isn't the mutual TLS between the `signer` and `server`?

Comment (cyli) [inline CHANGELOG.md]: There is mutual TLS between the signer and server, but we recently added support for the client and server to also support mutual TLS.

Comment (HuKeping) [inline CHANGELOG.md]: Haven't been aware that, it's cool though!

Comment (cyli) [inline CHANGELOG.md]: It was this PR:  https://github.com/docker/notary/pull/545/files#diff-1409503b7b31865d44186c540b70fb43R450

Comment (HuKeping) [inline CHANGELOG.md]: Thanks for the detail @cyli

**Comment by endophage** [comment]
LGTM

**Pull request merged**